auth/src/main/java/com/firebase/ui/auth/ui/email/RecoverPasswordActivity.java

```diff
@@ -70,12 +71,20 @@ private void next(final String email) {
                     @Override
                     public void onSuccess(Void aVoid) {
                         mActivityHelper.dismissDialog();
-
-                        Intent confirmIntent = ConfirmRecoverPasswordActivity.createIntent(
-                                RecoverPasswordActivity.this,
-                                mActivityHelper.getFlowParams(),
-                                email);
-                        startActivityForResult(confirmIntent, RC_CONFIRM);
+                        new AlertDialog.Builder(RecoverPasswordActivity.this,
+                                                R.style.FirebaseUI_Dialog)
+                                .setTitle(R.string.title_confirm_recover_password_activity)
+                                .setMessage(String.format(
+                                        getString(R.string.confirm_recovery_body),
+                                        email))
+                                .setOnDismissListener(new DialogInterface.OnDismissListener() {
+                                    @Override
+                                    public void onDismiss(DialogInterface anInterface) {
+                                        finish(RESULT_OK, new Intent());
+                                    }
+                                })
+                                .setPositiveButton(android.R.string.ok, null)
+                                .show();
```

Comment:
This is great.  Not sure why it was ever an Activity.  It may be better to use a DialogFragment rather than an AlertDialog (to survive rotations) but I am fine to leave that as a TODO.